packages/framework/src/Form/AutoCardGroupableExtension.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Form;
+
+use Override;
+use Shopsys\FormTypesBundle\ActionBarType;
+use Symfony\Component\Form\AbstractTypeExtension;
+use Symfony\Component\Form\Extension\Core\Type\FormType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+
+final class AutoCardGroupableExtension extends AbstractTypeExtension
```

Comment:
should it be final? What if they want to add another type to invisibleTypes on a project?